### DISCUSSION TITLE
Saving Space due to size of models

### DISCUSSION THREAD

**Discussion by javaarchive** [keras-team/autokeras#1585] [Q&A]
Recently I got a warning about low disk space and the only thing resource intensive I was running was an autokeras image classifier training. I checked disk usage and it seemed like the model folders were taking up **a lot** of storage.
![image](https://user-images.githubusercontent.com/20248577/120934097-cc0f5500-c6b1-11eb-8032-bdcec98861e6.png)
Is there any way I can reduce the size of the directories, perhaps maybe by archiving all of the epoch checkpoints but the last best one, I'm not sure if this would be safe to do. 

I'll consider setting a max model size but for now I don't have any constraints on the model size, a 1gb model is fine for me.

**Comment by haifeng-jin** [comment]
This is a problem in KerasTuner, we are fixed it. Will be fixed in the release of AutoKeras after the next KerasTuner release.
